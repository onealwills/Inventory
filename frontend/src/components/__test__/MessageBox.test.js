import { render } from "@testing-library/react";
import MessageBox from "../MessageBox";

test("<MessageBox/> component snapshot test", () => {
  const component = render(
    <MessageBox type="danger">yeye dey smell</MessageBox>
  );
  expect(component.container).toMatchSnapshot();
});
