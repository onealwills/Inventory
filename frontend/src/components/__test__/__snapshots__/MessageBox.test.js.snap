// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MessageBox/> component snapshot test 1`] = `
<div>
  <div
    class="alert alert-danger "
  >
    yeye dey smell
  </div>
</div>
`;
